fortes_webservice_wrapper/wrapper/fortes_ws.py

```diff
@@ -0,0 +1,33 @@
+from ..models.base import TClient, TFornecedor
+from .soap import BaseSoapWrapper
+
+
+class FortesWrapper(BaseSoapWrapper):
+
+    # def trata_retorno(self, data):
+    #     mensagem_retorno = data.find("Retorno")
+    #     mensagem_erro = data.find("Erro")
+    #
+    #     mensagem_retorno = data[mensagem_retorno:mensagem_erro]
+    #     mensagem_erro = data[mensagem_erro:]
+    #
+    #     return mensagem_retorno, mensagem_erro
+
+    def incluir_cliente(self, cliente):
+        data = TClient(**cliente)
+        r = self.get_client().service.IncluirClienteComJSON(data.json())
+        return r
+
+    def incluir_fornecedor(self, fornecedor):
+        data = TFornecedor(**fornecedor)
+        r = self.get_client().service.IncluirFornecedorComJSON(data.json())
+        return r
+
+    def excluir_cliente(self, document):
+        r = self.get_client().service.ExcluiClienteWithCNPJCPF(document)
```

Comment:
Todos os outros estão com `Com`, apenas por padronização:
```suggestion
        r = self.get_client().service.ExcluiClienteComCNPJCPF(document)
```

Reply:
Esses nomes que chamo depois do `.service` são os  métodos do `Fortes`, aí não posso alterar :crying_cat_face:

Reply:
> Esses nomes que chamo depois do `.service` são os métodos do `Fortes`, aí não posso alterar :crying_cat_face:

Entendi.